tests/cephfs/snapshot_clone/snap_schedule_retention_vol_subvol.py

```diff
@@ -420,6 +470,355 @@ def snap_retention_test(snap_test_params):
+        client, snap_test_params["path"], ret_val=snap_test_params["retention"]
+    )
+    log.info("Perform schedule snaps cleanup")
+    snap_util.sched_snap_cleanup(client, snap_path)
+    snap_util.remove_snap_schedule(client, snap_test_params["path"])
+    out, rc = client.exec_command(
+        sudo=True, cmd=f"rmdir {snap_path}/.snap/manual_test_snap"
+    )
+    log.info("Verify retention when mds_max_snaps_per_dir set to non-default value - 8")
+
+    log.info("Enable mgr debug logs")
+    cmd = "ceph config set mgr log_to_file true"
+    out, rc = client.exec_command(sudo=True, cmd=cmd)
+    log.info("Note current value for mds_max_snaps_per_dir")
+    out, rc = client.exec_command(
+        sudo=True, cmd="ceph config get mds mds_max_snaps_per_dir"
+    )
+    default_retention = int(out.strip())
+    log.info("Set mds_max_snaps_per_dir as 8")
+    out, rc = client.exec_command(
+        sudo=True, cmd="ceph config set mds mds_max_snaps_per_dir 8"
+    )
+    snap_util.create_snap_schedule(snap_test_params)
+    log.info("Verify 8 scheduled snaps are retained")
+    duration_min = 8
+    log.info(f"add sched data params : {client} {io_path} {duration_min}")
+    fs_util.run_ios_V1(client, io_path, run_time=duration_min)
+    wait_retention_check = 60
+    log.info(
+        f"Wait for additional time {wait_retention_check}secs to validate retention"
+    )
+    time.sleep(wait_retention_check)
+    if snap_util.validate_snap_retention(client, snap_path, snap_test_params["path"]):
+        test_fail = 1
+        log.error("A check to verify only 8 scheduled snaps retained, has failed")
+    log.info("Note last entry in mgr logs")
+    log.info("Fetch the FSID of the ceph cluster")
+    out, _ = installer_node.exec_command(sudo=True, cmd="cephadm ls")
+    data = json.loads(out)
+    fsid = data[0]["fsid"]
+    cmd = f"cd /var/log/ceph/{fsid}/;ls *mgr*log"
+    out, _ = installer_node.exec_command(sudo=True, cmd=cmd)
+    mgr_file = out.strip()
+    cmd = f"cat /var/log/ceph/{fsid}/{mgr_file}|grep Traceback|wc -l"
+    traceback_before, _ = installer_node.exec_command(sudo=True, cmd=cmd)
+    cmd = f"cat /var/log/ceph/{fsid}/{mgr_file}|grep Exception|wc -l"
+    exception_before, _ = installer_node.exec_command(sudo=True, cmd=cmd)
+    log.info(
+        f"Traceback info before: {traceback_before},Exception info : {exception_before}"
+    )
+    log.info("Try manual snap creation")
+    try:
+        if "subvol" in snap_test_params.get("test_case"):
+            snapshot = {
+                "vol_name": snap_test_params["fs_name"],
+                "subvol_name": snap_test_params["subvol_name"],
+                "snap_name": "manual_test_snap1",
+                "group_name": snap_test_params["group_name"],
+            }
+            fs_util.create_snapshot(client, **snapshot)
+        else:
+            cmd = f"mkdir {snap_path}/.snap/manual_test_snap1"
+            out, rc = client.exec_command(sudo=True, cmd=cmd)
```

Comment:
please add a logger for successful creation of manual snapshot.

Reply:
manual snapshot creation is not expected in this case, message been logged for failure